pkgs/development/python-modules/nix-build-profiler/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
```

Comment:
Can you break this off into a separate PR? IIUC it is unrelated other than being used to debug the creation of this PR. Feel free to tag me for review.